### PR TITLE
Change docker logging level to info on GCI configure script

### LINKED ISSUE #91223: GCI Docker Logs should be at info level
**What happened**:

Docker logs on GCI are set at the warning level. This makes hard to debug docker related issues due to a lack of logs. Log level should be set to info. 

**What you expected to happen**:

Log level is set to info. 

**How to reproduce it (as minimally and precisely as possible)**:

n/a

**Anything else we need to know?**:

n/a

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by bobbypage**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Docker logs on GCI are set at the warning level. This makes it hard to debug docker related issues due to a lack of detailed logs. Log level should be set to info.

**Which issue(s) this PR fixes**:

Fixes #91223

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1338,7 +1338,7 @@ function assemble-docker-flags {
   if [[ "${TEST_CLUSTER:-}" == "true" ]]; then
     docker_opts+=" --log-level=debug"
   else
-    docker_opts+=" --log-level=warn"
+    docker_opts+=" --log-level=info"
```
Comment (liggitt) [inline cluster/gce/gci/configure-helper.sh:1341]: what is the increase in log volume due to this?

Comment (cheftako) [reply cluster/gce/gci/configure-helper.sh:1341]: Have you looked at https://github.com/kubernetes/kubernetes/issues/27584? We used to have it at info and changed it to warn.

Comment (bobbypage) [reply cluster/gce/gci/configure-helper.sh:1341]: Thanks @liggit and @cheftako. There's some more info in #15590 and #15728. Those issues pretty old (from 2016), so let me take a closer look at current docker logging volume at info compared to warn today.

**Pull request closed without merging**